conda/recipes/cucim/meta.yaml

```diff
@@ -1,100 +0,0 @@
-# Copyright (c) 2021-2023, NVIDIA CORPORATION.
-
-{% set version = environ['RAPIDS_PACKAGE_VERSION'].lstrip('v') %}
-{% set py_version = environ['CONDA_PY'] %}
-{% set cuda_version = '.'.join(environ['RAPIDS_CUDA_VERSION'].split('.')[:2]) %}
-{% set cuda_major = cuda_version.split('.')[0] %}
-{% set date_string = environ['RAPIDS_DATE_STRING'] %}
-
-
-package:
-  name: cucim
-  version: {{ version }}
-
-source:
-  path: ../../..
-
-build:
-  number: {{ GIT_DESCRIBE_NUMBER }}
-  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
-  ignore_run_exports_from:
-    - {{ compiler('cuda') }}
-    {% if cuda_major != "11" %}
-    - cuda-cudart-dev
-    {% endif %}
-  script_env:
-    - AWS_ACCESS_KEY_ID
-    - AWS_SECRET_ACCESS_KEY
-    - AWS_SESSION_TOKEN
-    - CMAKE_C_COMPILER_LAUNCHER
-    - CMAKE_CUDA_COMPILER_LAUNCHER
-    - CMAKE_CXX_COMPILER_LAUNCHER
-    - CMAKE_GENERATOR
-    - PARALLEL_LEVEL
```

Comment:
Also should we still set `PARALLEL_LEVEL`?

Looks like it was dropped from the new recipe